packages/shelter-ui/src/slider.tsx.scss

```diff
@@ -0,0 +1,147 @@
+.scontainer {
+  position: relative;
+  display: flex;
+  align-items: center;
+
+  width: 100%;
+  margin-bottom: 16px;
+
+  height: 54px;
+
+  font-size: 10px;
+  color: var(--header-secondary);
+}
+
+.srange {
+  appearance: none;
+  -webkit-appearance: none;
+
+  background: transparent;
+  cursor: pointer;
+
+  width: 100%;
+
+  --upper-half: 0%;
+
+  z-index: 2;
+
+  /* This CSS is horrifically long but apparently they cannot be comma-separated */
```

Comment:
You can use a SCSS mixin to clean this up like this
```scss
@mixin track {
    appearance: none;
    -webkit-appearance: none;

    background: linear-gradient(to right, var(--brand-experiment) var(--upper-half), var(--brand-experiment) var(--upper-half), var(--primary-500) var(--upper-half), var(--primary-500) 100%);
    height: 8px;

    border-radius: 3px;
}

&::-ms-track { @include track; }
&::-webkit-slider-runnable-track { @include track; }
&::-moz-range-track { @include track; }
```

But I'll probably do that after merge if you don't so /shrug